db/data_migrations/20220909093819_fix_description_for_coffins.rb

```diff
@@ -0,0 +1,42 @@
+Sequel.migration do
+  # IMPORTANT! Data migrations should be Idempotent, they may get re-run as part
+  # of data rollbacks
+
+  up do
+    if TradeTariffBackend.uk?
+      Sequel::Model.db[:goods_nomenclature_descriptions_oplog]
```

Comment:
```suggestion
      GoodsNomenclatureDescription::Operation.where(goods_nomenclature_sid: 107510, goods_nomenclature_description_period_sid: 155818, filename: 'tariff_dailyExtract_v1_20220816T235959.gzip').update(description: 'Coffins')
```

Reply:
This is a concious choice - I'm avoiding loading the data model itself - generally in migrations its best to keep as low level as possible and just stick to a rubyized SQL equivalent